app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/address/disposal_price.scala.html

```diff
@@ -73,40 +84,40 @@
 
     @caption(addressCaption)
 
-    @formWrapper(address.routes.PropertyDetailsController.disposalPriceSubmit()) {
-      @govukInput(Input(
-        id = key,
-        name = key,
-        value = form.data.get(key),
-        label = Label(
-          content = Text(title),
-          isPageHeading = true,
-          classes = "govuk-label--xl"
-        ),
-        classes = "govuk-input--width-10",
-        hint = Some(Hint(
-          content = HtmlContent(messages(s"$key$userKey.helpText"))
-        )),
-        prefix = Some(PrefixOrSuffix(
-          content = Text("£")
-        )),
-        errorMessage = form.error(key).map(e => ErrorMessage(
-          content = Text(messages(s"$key$userKey.${e.message}")),
-          visuallyHiddenText = Some(messages("generic.error"))
-        )),
-        autocomplete = Some(AutoCompleteType.On.value)
-      ))
-
-       @details(
+    <h1 class="govuk-heading-xl">@title</h1>
+
+    <p class="govuk-body" id="multipleDisposalsDisposalPrice-hint">@messages(s"$key$userKey.helpText")</p>
+
+    @details(
         messages(s"$key.link"),
         Html(
-          s"""
+            s"""
           | <p class="govuk-body">${messages(s"$key.details.p1")}</p>
           |""".stripMargin
         )
-      )
+    )
+
+    @formWrapper(address.routes.PropertyDetailsController.disposalPriceSubmit()) {
+        @govukInput(Input(
+            label = Label(
+                content = Text(input),
```

Comment:
please can we call this 'label'?

Reply:
Just saw your discussion with Lukasz about the 'label'. You still want me to change this from 'input' to 'label'?

Reply:
Just the variable name 'input' -> 'label'